.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish snapshot
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-snapshot:
+    runs-on: ubuntu-latest
+    if: github.repository == 'square/anvil'
+    timeout-minutes: 25
+
```

Comment:
Now that we don't have a dependency on
```
 - test-ubuntu
  - test-gradle-plugin
```
      
That means we'll be publishing a snapshot even if one of those fails right?

Reply:
Sort of, but that's what required checks and "Require branches to be up to date before merging" are for.  Our `main` branch can never be red unless an admin intentionally bypasses those restrictions.

With that said, the `ci.yml` workflow is still triggered by pushes to `main`, so if something is broken we'll still get notified of it.

Reply:
That's fair 👍